### DISCUSSION TITLE
Copilot chat window in Android Studio is messed up

### DISCUSSION THREAD

**Discussion by Matt8Black** [community/community#124897] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The conversation with Copilot is messed up sometimes in Android Studio. The response have half the code in code block and the other half in normal text, and half the text is normal and half of it in code block
<img width="996" alt="Screenshot 2024-05-21 at 7 46 45 AM" src="https://github.com/community/community/assets/78480170/1c126e1b-4a44-4fe7-a7cb-5682f1a079bd">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by angelo-manalo** [comment]
1. **Update Copilot Plugin:**
   - Go to `File > Settings > Plugins`.
   - Search for `GitHub Copilot`.
   - Click `Update` if available.

2. **Reinstall Copilot Plugin:**
   - Go to `File > Settings > Plugins`.
   - Search for `GitHub Copilot` and uninstall it.
   - Restart Android Studio.
   - Reinstall the GitHub Copilot plugin.

3. **Toggle Copilot:**
   - Click the Copilot icon in the toolbar to disable and then re-enable it.

4. **Clear Cache:**
   - Go to `File > Invalidate Caches / Restart`.
   - Select `Invalidate and Restart`.
